sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/HyperLogLogPlusPlus.scala

```diff
@@ -48,6 +46,8 @@ import org.apache.spark.sql.types._
  * @param relativeSD the maximum estimation error allowed.
  */
 // scalastyle:on
+@ExpressionDescription(
+  usage = "_FUNC_(expr, relativeSD) - Returns the estimated cardinality by HyperLogLog++.")
```

Comment:
Also provide the usage of `_FUNC_(expr)` (it will mention the default value of relativeSD). Can we explain relativeSD at here?